PulseEco/Info.plist

```diff
@@ -76,5 +76,7 @@
 	</array>
 	<key>UIUserInterfaceStyle</key>
 	<string>Light</string>
+	<key>NSLocationWhenInUseUsageDescription</key>
+	<string>This app uses location services to show information more relevant to the user according to his location.</string>
```

Comment:
This is a bit unclear, it should specify the purpose of the location request 
Use this instead
This app uses location services to provide air quality, pollution levels, and other information based on your current location.